style.css

```diff
@@ -515,4 +598,107 @@ button:hover {
   .custom-gap {
     gap: 5rem;
   }
+
+  #popup {
+    right: 194px;
+    position: fixed;
+    width: 1108px;
+    height: 678px;
+    background-color: white;
+    top: 21px;
+    display: none;
+  }
+
+  .popup-active {
+    display: block !important;
+  }
+
+  .popup-name {
+    position: relative;
+    left: 129px;
+  }
+
+  .popup-img {
+    display: block;
+    width: 847px;
+    height: 396px;
+    position: relative;
+    right: -128px;
+  }
+
+  .p-popup {
+    text-align: justify;
+    width: 442px;
+    position: relative;
+    left: 119px;
+    float: left;
+    height: 171px;
+  }
+
+  .popup-list {
+    gap: 11px;
+    float: right;
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    width: 260px;
+    position: relative;
+    left: -214px;
+    column-gap: 8px;
+  }
+
+  .popup-list-item {
+    background: #ebebff;
+    border-radius: 8px;
+    color: #6070ff;
+    border: none;
+    padding: 5px 10px;
+  }
+
+  .popup-list-item-hidden {
+    background: #ebebff;
+    border-radius: 8px;
+    color: #6070ff;
+    border: none;
+    padding: 5px 10px;
+    display: inline-block;
+  }
+
+  .button-from {
+    position: relative;
+    bottom: -133px;
+    text-align: center;
+    display: flex;
+    gap: 5px;
+    left: 20%;
+    top: 15%;
+  }
+
+  .popup-button {
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    border: 2px solid #396df2;
+    padding: 10px;
+    color: #396df2;
+    background-color: transparent;
+  }
+
+  .close {
+    position: relative;
+    left: 88%;
+    font-size: 29px;
+    display: inline-block;
+    transform: rotate(90deg);
+    font-weight: 500;
+    cursor: pointer;
+    user-select: none;
+  }
+
+  .popup-image-mobile {
+    display: none;
+  }
+
+  /* .popup-img {
+    display: block;
+  } */
```

Comment:
- [ ] <b>REQUIRED:</b> Keep your code clean by making sure to remove all commented-out code.
- [ ] <b>REQUIRED:</b> Lastly, Kindly make sure that your popup interface looks exactly like the provided design.

Reply:
> * [x]  REQUIRED: Keep your code clean by making sure to remove all commented-out code.
> * [ ]  REQUIRED: Lastly, Kindly make sure that your popup interface looks exactly like the provided design.

Your popup window in mobile view overflows the container, you can fix this using media queries. In the desktop version, it does not load the correct project, this is because the projects are not dynamically created from javascript. Kindly fix this too